pkg/setup/build_crd.go

```diff
@@ -38,7 +38,8 @@ func buildBlockDeviceCRD() (*apiext.CustomResourceDefinition, error) {
 		WithPrinterColumns("Size", "string", ".spec.capacity.storage").
 		WithPrinterColumns("ClaimState", "string", ".status.claimState").
 		WithPrinterColumns("Status", "string", ".status.state").
-		WithPrinterColumns("Age", "date", ".metadata.creationTimestamp")
+		WithPrinterColumns("Age", "date", ".metadata.creationTimestamp").
+		WithPrinterColumns("FSType", "string", ".spec.Filesystem.Type")
```

Comment:
use `WithPriorityPrinterColumns` as we need the fstype to be displayed only when we do a `kubectl get bd -o wide`.

Its a convention to keep the age of resource as the last column. Therefore, we can move fstype after the `Path` column